lib/inspec/plugin/v2/plugin_types/reporter.rb

```diff
@@ -0,0 +1,38 @@
+require_relative "../../../run_data"
+
+module Inspec::Plugin::V2::PluginType
+  class Reporter < Inspec::Plugin::V2::PluginBase
```

Comment:
Since this PR was raised, the base reporter class had some option handling added to it; it would be great to copy that here so that the options will also work with plugin reporters

https://github.com/inspec/inspec/commit/2103b52dbea8224f0c96d02912ec6cc940d27520#diff-bc3d043c1864e7c4d39d9aec04707b34R13